modular_skyrat/modules/modular_implants/code/nif_research.dm

```diff
@@ -18,3 +18,13 @@
 	category = list(RND_CATEGORY_EQUIPMENT)
 	departmental_flags = DEPARTMENT_BITFLAG_CARGO
 
+/datum/design/soulcatcher_device
+	name = "Soulcatcher device"
+	desc = "PLACEHOLDER TEXT"
```

Comment:
An RSD instrument that lets the user pull the consciousness from a body and store it virtually.